pkgs/applications/misc/electrum/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, fetchFromGitHub, python3, python3Packages, zbar, secp256k1
+{ stdenv, fetchurl, fetchFromGitHub, qt5, python3, python3Packages, zbar, secp256k1
 , enableQt ? !stdenv.isDarwin
```

Comment:
Is there any purpose left for `enableQt`? Seems like some of what is done here needs to be conditional on this flag, if it is to be retained.

Reply:
It brings in using `pyqt`.

Reply:
Conditional now.